views/apps.ejs

```diff
@@ -9,12 +9,22 @@
 
 
     <!-- scripts -->
-    <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.24/browser.js"></script>
```

Comment:
may I know, why we are reverting back to browser.js instead of using babel standalone ?